prov/hook/trace/src/hook_trace.c

```diff
@@ -830,6 +836,110 @@ static struct fi_ops_tagged trace_tagged_ops = {
 	.injectdata = trace_tinjectdata,
 };
 
+static int trace_setname(fid_t fid, void *addr, size_t addrlen)
+{
+	int ret = 0;
+	struct hook_ep *myep = NULL;
+
+	if (fid->fclass == FI_CLASS_EP) {
```

Comment:
Could FI_CLASS_PEP also be handled here? same for getname.

Reply:
Yes, I can enable trace cm for FI_CLASS_PEP.  The current functions are set up for FI_CLASS_EP only.